packages/sdk/src/sound/setSoundStateOptions.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Parameters to the `Sound.setSoundState` and `Actor.playSound` calls.
+ */
+export type SetSoundStateOptions = {
+    /**
+     * enabled (pause/resume).
+     */
+    enabled?: boolean,
+
+    /**
+     * pitch offset in halftones (0=default, 12=one octave higher, -12=one octave lower)
+     */
+    pitch?: number,
+
+    /**
+     * volume modifier, measured in DB (0=default,10=10x power, -10=1/10 power.
```

Comment:
>volume modifier, measured in DB [](start = 7, length = 31)

This isn't what I expected. Would prefer a 0-1 volume percentage, seems more intuitive.

Reply:
yeah, I was getting too creative here. Will restore to 0-1

---
In reply to: [256950001](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/191#discussion_r256950001) [](ancestors = 256950001)